lib/src/widgets/links_card_widget/links_card_button.dart

```diff
@@ -18,7 +18,7 @@ class LinksCardButton extends StatefulWidget {
 class _LinksCardButtonState extends State<LinksCardButton> {
   bool _isHovered = false;
 
-  static const _iconSize = 13.0;
+  static const _iconSize = 10.0;
```

Comment:
Won't changes to this file affect the already implemented first widget variation?

Reply:
http://localhost:50307/#/

Reply:
<img width="373" alt="image" src="https://user-images.githubusercontent.com/40931732/187677466-352c4e64-89e2-4196-b3ef-1fd5b2c7a6ee.png">